.github/workflows/pr-merge-webhook.yml

```diff
@@ -10,20 +10,19 @@ on:
 
 jobs:
   create-docs-ticket:
-    # Only on merged PRs that contain the DocumentationNeeded flag
-    if: github.event.pull_request.merged && contains(github.event.pull_request.labels.*.name, 'DocumentationNeeded')
+    # Only on merged PRs
+    if: github.event.pull_request.merged
     runs-on: ubuntu-24.04
     steps:
       - uses: actions/checkout@v4
 
-      - name: Create Issues
-        id: create-issues
-        uses: actions/github-script@v7
-        with:
-          github-token: ${{ secrets.DOCS_ISSUES_PAT }}
-          script: |
-            const script = require('.github/workflows/create-docs-issues.js');
-            await script({github, context});
+      - name: Trigger PR Merge Webhook
+        id: trigger-pr-merge-webhook
```

Comment:
Do we need to assign this id? I see we were before, but I find it a bit of an anti-pattern unless you actually need to refer to it later by id.

Reply:
No, I don't think we do. I'll remove.